recipes/tsid/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "tsid" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stack-of-tasks/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0180b23611f62a5e9539422702d5e21d6db2a277c0e9cdb3d282755f07addb70
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - cmake
+    - make  # [not win]
+  host:
+    - eigenpy
```

Comment:
You need python in `host` and `run` so it builds once per python version as they have incompatible ABIs